presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -94,28 +85,25 @@ public void testDropTable()
 
     @Test
     public void testViews()
-            throws SQLException
     {
-        execute("CREATE OR REPLACE VIEW tpch.test_view AS SELECT * FROM tpch.orders");
+        mysqlServer.execute("CREATE OR REPLACE VIEW tpch.test_view AS SELECT * FROM tpch.orders");
```

Comment:
Let's leave the `execute` method, at least for now, as it makes the diff easier to read.